benchexec/tablegenerator/react-table/src/components/ScatterPlot.js

```diff
@@ -37,6 +44,11 @@ const resultsOptions = {
   correct: "Correct only",
 };
 
+const regressionOptions = {
+  hidden: "Hidden",
```

Comment:
I would prefer something like `None`. Actually, with just enabled/disable I would prefer a check box, but as far as I understood we could trivially add non-linear options, so let's do that and keep the drop down.